### PR TITLE
VBM: feature/drawer-menu-content

### PR DESCRIPTION
**Pull request by Val-M9**
[Відео без назви - Screen Recording - 12.09.2022, 18_44_53.webm](https://user-images.githubusercontent.com/71255723/189699516-c00dbb77-2d00-48a7-a68f-25537b1fb0f3.webm)

### CODE REVIEW

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import { ColorPalette, DEFAULT_LOCALE } from '@vse-bude/shared';
+import { notification } from '~/services/services';
+import { TouchableOpacity, Text, View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const LanguageButtons: FC = () => {
+  const [currentLang, setCurrentLang] = useState(i18next.language);
+  const handleChangeLanguage = (lang: string) => {
+    i18next.changeLanguage(lang, (err) => {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:12]:
```suggestion
  const [currentLang, setCurrentLang] = useState(i18next.language);
  const { i18n } = useTranslation();

  const handleChangeLanguage = (lang: string) => {
    i18n.changeLanguage(lang, (err) => {
```

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React, { FC, useState } from 'react';
+import i18next from 'i18next';
+import { ColorPalette, DEFAULT_LOCALE } from '@vse-bude/shared';
+import { notification } from '~/services/services';
+import { TouchableOpacity, Text, View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const LanguageButtons: FC = () => {
+  const [currentLang, setCurrentLang] = useState(i18next.language);
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:10]: You don't need additional state here

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  const handleChangeLanguage = (lang: string) => {
+    i18next.changeLanguage(lang, (err) => {
+      setCurrentLang(lang);
+      if (err) {
+        notification.error(err);
+      }
+    });
+  };
+  const isUa = currentLang == DEFAULT_LOCALE;
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:19]:
```suggestion
  const isUa = currentLang === DEFAULT_LOCALE;
```

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+  return (
+    <View
+      style={[
+        globalStyles.mr4,
+        globalStyles.flexDirectionRow,
+        globalStyles.justifyContentEnd,
+      ]}
+    >
+      <TouchableOpacity
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:29]: Try to refactor it to something like this:
```
{i18n.languages.map(lang => {
  const isCurrent = i18n.resolvedLanguage === lang;

  return (
   <TouchableOpacity style={[styles.languageButton, isCurrent && styles.current]}>
     <Text>{lang}</Text>
   </TouchableOpacity>
  );
})}
```

File: packages/mobile/src/navigation/drawer/drawer.navigation.tsx [hunk 5]
```diff
@@ -15,7 +16,10 @@ const HomeWithMenuNavigation: FC = () => {
   };
 
   return (
-    <Drawer.Navigator screenOptions={options}>
+    <Drawer.Navigator
+      screenOptions={options}
+      drawerContent={() => <DrawerContent />}
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/drawer/drawer.navigation.tsx:21]:
```suggestion
      drawerContent={DrawerContent}
```

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+import { notification } from '~/services/services';
+import { TouchableOpacity, Text, View } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const LanguageButtons: FC = () => {
+  const [currentLang, setCurrentLang] = useState(i18next.language);
+  const handleChangeLanguage = (lang: string) => {
+    i18next.changeLanguage(lang, (err) => {
+      setCurrentLang(lang);
```
Comment (mazurGit) [inline packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:13]: We also need to change language for dayjs

File: packages/mobile/src/navigation/drawer/drawer.navigation.tsx (hunk 5 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/drawer.navigation.tsx:21]: fixed

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx (hunk 4 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:29]: done

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx (hunk 3 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:19]: removed

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx (hunk 1 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:12]: fixed

File: packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx (hunk 2 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/drawer/components/language-buttons/language-buttons.tsx:10]: fixed

**Review by AleksandrTermenzhy — APPROVED** [review]

**Review by VladimirBludov — APPROVED** [review]

**Pull request merged**